test/Orleans.Transactions.Azure.Test/TestFixture.cs

```diff
@@ -33,5 +34,14 @@ public ISiloHostBuilder CreateSiloBuilder(string siloName, ClusterConfiguration
                     .UseTransactionalState();
             }
         }
+
+        public static void CheckForAzureStorage()
```

Comment:
I highly recommend that you use BaseAzureTestClusterFixture (or a similar approach). The way you are using it here will only skip the test after it attempted to start the test cluster (and will possible throw an exception and fail the test, instead of marking it as skipped).
The gist of it is that the fixture should not try to create a cluster if it cannot use azure storage (but without throwing), and then the test class itself will just call `fixture.EnsurePreconditionsMet()` throw that it's correctly skipped

Reply:
> I highly recommend that you use BaseAzureTestClusterFixture 

BaseAzureTestClusterFixture is in tester which pulls in an unreasonable number of dependencies.  I shouldn't need to use tester to write test for extension libraries.

> The way you are using it here will only skip the test after it attempted to start the test cluster

I don't mind making it part of fixture, but the pattern I'm using throws during the test class construction, which is complete before the cluster is created, correct?

> So in theory it is separable from the StorageEmulator class anyway.

Logically separable, yes, but dependent on it and related to our storage emulator test support, which is in src/OrleansTestingHost/Utils/StorageEmulator.cs.

I don't think this adds (relatively) any meaningful risk to test host, but I'll remove the call all the same.